apps/system/js/notifications.js

```diff
@@ -137,7 +137,7 @@ var NotificationScreen = {
             }
             break;
           case 'desktop-notification-close':
-            this.removeNotification(detail.id);
+            this.removeNotification(detail.id, detail.appName);
```

Comment:
1) It's not clear how it's expected to work, if signature of function 'removeNotification()' still have only 1 parameter?
2) it seems that 'appName' field will never be provided in notification details, at least in current version of code